### DISCUSSION TITLE
可以公开下sign的计算方法吗？

### DISCUSSION THREAD

**Discussion by M1saka10010** [AlistGo/alist#2740] [Q&A] [answered]
尝试地读了下代码好像是base64(sha256(文件名:时间)):时间，但是那个SecretKey是啥也没找到，能请作者大大公开下算法吗？我想自己实时计算key用来反代

**Comment by xhofe** [comment, marked as answer]
https://alist.nn.ci/guide/drivers/common.html#download-proxy-url

**Reply by M1saka10010** [reply]
抱歉，我瞎了没看到
